var/spack/repos/spack_repo/builtin/packages/iotaa/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
+
+from spack.build_systems.python import PythonPipBuilder
+from spack.package import *
+
+
+class Iotaa(PythonPackage):
+    """A simple workflow engine with semantics inspired by Luigi
+    and tasks expressed as decorated Python functions"""
+
+    pypi = "iotaa/iotaa-1.2.3-py3-none-any.whl"
+
+    maintainers("maddenp-noaa")
```

Comment:
```suggestion
    maintainers("maddenp")
```
Note the project URL: https://github.com/maddenp/iotaa